src/config/sidenav/components.js

```diff
@@ -21,5 +21,6 @@ export default t => ({
     { to: '/components/textinput/', label: t('textinput:title') },
     { to: '/components/searchinput/', label: t('searchinput:title') },
     { to: '/components/icon/', label: t('icon:title') },
+    { to: '/components/block/', label: t('block:title') },
```

Comment:
Is the order of these meaningful? I thing it might be a good idea to use e.g. alphabetical order.

Reply:
It's the order of appearance I think, so no real meaning behind it. Might be a good idea to arrange them. I can do it in this PR as well if need be or it can be done separately.

Reply:
Either way is fine, not in the scope of this PR really. E.g. alphabetical sort like with styleguidist in ui-components would be good? Not too big thing to change I guess, so why not do it now?